core/src/main/java/org/infinispan/configuration/cache/CacheMode.java

```diff
@@ -5,6 +5,11 @@
  */
 public enum CacheMode {
    /**
+    * Data is not replicated and certain features are not possible
```

Comment:
@rvansa My main reason to not make SIMPLE a cache mode is that the current definition of what a SIMPLE cache seems very closely tied to the needs of Hibernate. For someone else, I'm not sure the trade-offs will be the same, for instance why support eviction but not persistence? So I would say at the very least it needs a little bake-in time before we expose it on the same level as LOCAL.